tests/tests_default.yml

```diff
@@ -2,6 +2,54 @@
 ---
 - name: Ensure that the role runs with default parameters
   hosts: all
-  gather_facts: false  # test that role works in this case
-  roles:
-    - linux-system-roles.template
+  gather_facts: true  # test that role works in this case
+  become: yes
+  become_user: root
+
+  tasks:
+    - name: Test default settings
+      block:
+        - name: Run postgresql role
+          include_role:
+            name: linux-system-roles.postgresql
```

Comment:
For it to work with `tox`, and in our CI, you must do `mv tests/roles/linux-system-roles.template/ tests/roles/linux-system-roles.postgresql`. `tests/roles/linux-system-roles.postgresql` contains symlinks to the role so that it can be run from tests.